includes/functions/llms.functions.log.php

```diff
@@ -131,17 +131,17 @@ function llms_get_callable_name( $callable ) {
 
 	if ( is_array( $callable ) && ! empty( $callable ) ) {
 
-		// Class and class method: [ $class, 'method' ].
+		// Class and class method: [ $class, 'method' ]. (phpcs:ignore Squiz.PHP.CommentedOutCode.Found).
 		if ( is_object( $callable[0] ) ) {
 			return get_class( $callable[0] ) . '->' . $callable[1];
 		}
 
-		// Static class + method: [ 'class', 'method' ].
+		// Static class + method: [ 'class', 'method' ]. (phpcs:ignore Squiz.PHP.CommentedOutCode.Found).
```

Comment:
The method is static, not the class.
The example should be a string, not an array.
```suggestion
		// A class name and a static method name: 'class' . '::' . 'method'.
```
Changes the percentage of tokens that look like code from 42% to 24%, which is under the default 40% threshold.